graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/fetchers/deployed/MutationFetcher.java

```diff
@@ -48,4 +57,28 @@ protected Parameters getDatastoreParameters(DataFetchingEnvironment environment)
           .build();
     }
   }
+
+  protected Collection<ValueModifier> buildModifiers(
+      EntityModel entityModel,
+      Keyspace keyspace,
+      Predicate<String> hasArgument,
+      Function<String, Object> getArgument) {
+
+    List<ValueModifier> modifiers = new ArrayList<>();
+    for (FieldModel column : entityModel.getRegularColumns()) {
+      String graphqlName = column.getGraphqlName();
+
+      if (hasArgument.test(graphqlName)) {
+        Object graphqlValue = getArgument.apply(graphqlName);
+        modifiers.add(
+            ValueModifier.set(
+                column.getCqlName(), toCqlValue(graphqlValue, column.getCqlType(), keyspace)));
+      }
+    }
+    if (modifiers.isEmpty()) {
+      throw new IllegalArgumentException(
+          "Input object must have at least one non-PK field set for an update");
+    }
+    return modifiers;
+  }
```

Comment:
This is only called in `UpdateFetcher`. I think you started moving it when we were considering unifying processing between UPDATE and INSERT, but now it's not necessary to move it anymore.

Reply:
Yes, sorry for that. Reverted. I will do this is a separate PR.